scripts/ci/notification/matrix_notification.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -e
```

Comment:
Any reason not to use all the nice options?
```suggestion
set -euo pipefail
```

Reply:
Nope. I added it.